.github/workflows/test_and_release.yml

```diff
@@ -56,19 +56,36 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
+    - name: Install trame-vtk.js
+      run: |
+        mkdir -p ./trame_vtk/modules/common/serve
+        curl https://unpkg.com/vue-vtk-js@3.1.6 -Lo ./trame_vtk/modules/common/serve/trame-vtk.js
+
     - name: Install and Run Tests
       run: |
+        pip install --extra-index-url https://wheels.vtk.org vtk-osmesa
```

Comment:
This won't work. In the `tests/requirements.txt`, `pyvista` pulls normal `vtk` from PyPI.

Reply:
```suggestion
```